lib/telemetry.go

```diff
@@ -338,7 +338,7 @@ func InitTelemetry(cfg TelemetryConfig) (*metrics.InmemSink, error) {
 	metricsConf.BlockedPrefixes = cfg.BlockedPrefixes
 
 	var sinks metrics.FanoutSink
-	addSink := func(name string, fn func(TelemetryConfig, string) (metrics.MetricSink, error)) error {
+	addSink := func(fn func(TelemetryConfig, string) (metrics.MetricSink, error)) error {
```

Comment:
I couldn't find anywhere this name should be used. Looks like it was added in a WIP commit in a7038454fd317bd021068396ee681944bf8ce55e, so I guess probably safe to remove.